gflownet/gflownet.py

```diff
@@ -57,6 +53,7 @@ def __init__(
         sample_only=False,
         replay_sampling="permutation",
         train_sampling="permutation",
+        eval_config=None,
         **kwargs,
     ):
         # Seed
```

Comment:
Why is this used beyond logger's `use_context` ? -> modify docstring if it's for more things, or modify argument if it's just to set `use_context`

Reply:
Not sure right now to be honest. I need to look into it more carefully.

Reply:
I'm not sure the comment was clear, I meant the `active_learning` argument

Reply:
![Screenshot from 2024-02-19 15-03-16](https://github.com/alexhernandezgarcia/gflownet/assets/17404586/b06a5f52-8d83-4c52-a3f8-bc772a143f13)
GitHub can definitely improve this...

The `active_learning` flag is technically not deprecated but definitely not well maintained. So we can remove it too and deal with this later.

Reply:
Ok. Done in [`dbe2e51` (#287)](https://github.com/alexhernandezgarcia/gflownet/pull/287/commits/dbe2e51869146a7764f8c233375e7d35f217aa05)